doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,183 @@
+.. _committer-expectations:
+
+Committer Expectations
+######################
+
+Overview
+********
+
+The Zephyr project encourages you to submit changes as small pull requests.
```

Comment:
"Small" may not be the perfect word here but I reckon it's the easiest to pass the idea to an inexperienced user, people that don't do many code reviews have no clear idea of what "easily reviewable" means, and while it's explained right below, I think that the idea of "small" is a good enough starting point for readers that don't get to the details explanation and can't wait to hit the "Open PR" button. But to your point, the kernel documentation does not quite mention "small" in the context of a commit, though it sort-of does in term of a patchseries: https://docs.kernel.org/process/submitting-patches.html#split-changes

Reply:
> The "Trivial" label is used to mark PRs that are not subject to the 2 day review period.

Found it in doc/project/dev_env_and_tools.rst , thanks.

Reply:
I just realized most of the text that follows is about "smaller" PRs rather than "small" PRs that we've been struggling to define. This switch would IMHO resolve a few discussions below. In fact the word "smaller" is already used a few times.

```suggestion
The Zephyr project encourages you to submit changes as smaller pull requests.
```